example/tailwind.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  purge: ["./src/**/*.jsx", "./src/**/*.tsx"],
+  theme: {
+    extend: {
+      spacing: {
+        "1-1/2": "6px",
+        "13-1/2": "54px",
+        17: "68px",
```

Comment:
Is it possible to compute these off of the `theme`?

Reply:
@austin94 this is still an open question for me.

Reply:
I thought through this some and I'm leaning toward not computing these values right now. Some thoughts:

- There are some areas where we can reference values from the existing Tailwind theme. With the spacing values as string I'm not sure this would be a valid path here.
- For a standard consumer that is using Tailwind, they wouldn't include `react-theming` and it's required peer dependencies. 

With these values not being base-4, would it make more sense to inline the pixel values in the `*.module.css` files? It would cleanup the theme and make it obvious that these are one-off values.

Reply:
The latter proposal makes the most sense to me.

Reply:
> With the spacing values as string...

I was wondering if you could use polished `math` like we do in other places.

Reply:
> you could use polished math like we do in other places

This makes sense for Garden maintained areas, but for a 3rd party consumer or product this feels like it moves away from the advantages of using these tokens. This would lock the component to specific version of styled-components as a peer dependency.

I'll move it to local styles for now.

Reply:
> This would lock the component to specific version of styled-components as a peer dependency

I'm fine with the current approach. But I still don't understand your assertion. The only added dependency in the other proposal would be the addition of the `polished` utility. I wasn't suggesting adding `styled-components`.

Reply:
I thought you were referring to the `theme` provided from `react-theming`. Polished would be another approach using the Tailwind `theme`